chirp/drivers/ic2730.py

```diff
@@ -295,6 +296,7 @@ class IC2730Radio(icf.IcomCloneModeRadio):
     _can_hispeed = True
     _raw_frames = True
     _highbit_flip = True
+    _MUNCH_CLONE_RESP = True
```

Comment:
This flag was added for one specific radio that the author claimed needed this, but I was always skeptical. Especially with some of the changes I made recently to this module to fully parse the entire stream from the radio, I really don't think this should be necessary. With two other similiar-vintage raw-speaking icom radios working well (5100 and 4100) I'm really not wanting to do this unless there's a really good reason. Can you tell me why you think we need to add this? Like, I want to see logs of failures :)

Is it possible this was left-over from some debugging and isn't actually necessary?